nntrainer/tensor/blas_interface.cpp

```diff
@@ -1002,11 +1003,10 @@ void copy_s16_fp32(const unsigned int N, const int16_t *X, float *Y) {
 void copy_s16(const unsigned int N, const int16_t *X, int16_t *Y) {
 #ifdef USE_NEON
   nntrainer::neon::copy_s16(N, X, Y);
-#else
+#endif
```

Comment:
Please change this to if else clause to avoid duplicate copy.
```suggestion
#else
```

Reply:
fixed